client/components/token-field/suggestions-list.jsx

```diff
@@ -83,7 +83,6 @@ var SuggestionsList = React.createClass( {
 	},
 
 	_renderSuggestions: function() {
-
```

Comment:
Not a big deal, but I wouldn't make this modification here - I think general practice is to only make cleanups like this in code that's very close to areas where you're already making other changes.